crates/shared/src/http_solver_api/constants.rs

```diff
@@ -0,0 +1,13 @@
+use ethcontract::U256;
+
+lazy_static::lazy_static! {
+    // Estimates from multivariate linear regression here:
+    // https://docs.google.com/spreadsheets/d/13UeUQ9DA4bHlcy9-i8d4nSLlCxSfjcXpTelvXYzyJzQ/edit?usp=sharing
+    pub static ref GAS_PER_ORDER: U256 = U256::from(66_315);
```

Comment:
Nit: I think it is slightly nicer to use built in types like u64 and convert them with `.into()` to U256 when they are used. That way there is no lazy_static. But not sure what others prefer.